presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -88,7 +89,7 @@ public static Slice stPolygon(@SqlType(StandardTypes.VARCHAR) Slice input)
 
     @Description("Returns the area of a polygon using Euclidean measurement on a 2D plane (based on spatial ref) in projected units")
     @ScalarFunction("ST_Area")
-    @SqlType(StandardTypes.DOUBLE)
+    @SqlType(DOUBLE)
```

Comment:
It would be nice to do the static import change in another commit as that change modifies several other parts of this class.